core/modules/replica/protocol.proto

```diff
@@ -465,7 +466,7 @@ message ProtocolReplicaInfo {
 message ProtocolResponseReplicate {
 
     /// The completion status of the operation
-    required ProtocolStatus status = 1;
```

Comment:
That's exactly what has happened. I had to change the original `enum` from:
```
enum ProtocolStatus {
    SUCCESS       = 0;
    QUEUED        = 1;
    IN_PROGRESS   = 2;
    IS_CANCELLING = 3;
    BAD           = 4;
    FAILED        = 5;
    CANCELLED     = 6;
}
```
to the new one (note an extra status code `CREATED`) added here:
```
enum ProtocolStatus {
    CREATED       = 0;
    SUCCESS       = 1;
    QUEUED        = 2;
    IN_PROGRESS   = 3;
    IS_CANCELLING = 4;
    BAD           = 5;
    FAILED        = 6;
    CANCELLED     = 7;
}
```
That was my expectation that the change (if not taken care of) would break (in a bad way) the implementation. So, I just followed your earlier advise and changed indexes of the relevant fields in the dependent messages.